.github/workflows/main.yml

```diff
@@ -54,6 +54,10 @@ jobs:
           cat out.log
           grep -e "All .* tests passed" out.log || exit 1
 
+      - name: Docker Integration Test
```

Comment:
We could consider just using this to replace the above integration test, which only exists as a minimal test to ensure that the verifier actually works in the container.

Reply:
Sure, I'm ok with that - this will save a few minutes off of the `docker` build job in CI, and we're still running tests in the `build` job so we shouldn't miss much. Done.